addon/components/document-list-item.hbs

```diff
@@ -44,9 +44,10 @@
       </td>
     {{else if (eq column "link")}}
       <td>
-        {{#let (this.config.documentListLinkTo @document) as |linkConfig|}}
+        {{#let (await (this.config.documentListLinkTo @document)) as |linkConfig|}}
```

Comment:
Added option to return the whole url instead.
Had some problems with routing to outside of the engine with `LinkTo`

Reply:
Seemed problematic as it is hardcoded into the engine which external routes it expects

Reply:
I suggest adding a property `isExternal` to the result of `documentListLinkTo` and if `true` use the `<LinkToExternal />` component instead of the regular `<LinkTo />`

Reply:
also ran into a [bug](https://github.com/emberjs/ember.js/issues/19978) when the passed model is null, which would require an `if` if a model is passed or not. Just passing an url just seemed to address all these problems

Reply:
> Seemed problematic as it is hardcoded into the engine which external routes it expects

I think it's fine if we define something like `document-search-result` as an external route. Are host apps required to define them?

Reply:
Yes hosts are required to define all external routes which are in the engine config

Reply:
Damn, let's go with this solution then for the moment..